tabs/src/translations/en-US.json

```diff
@@ -1,6 +1,58 @@
 {
     "translation": {
 
+        "Query Runner": {
+            "Key": "Key",
+            "Value": "Value",
+            "Run query": "RUN QUERY",
+            "Request body": "Request Body",
+            "Request headers": "Request Headers",
+            "Response body": "Response Body",
+            "Response headers": "Response Headers",
+            "Response": "Response",
+            "Warning": "To use Graph requests, you must connect the app to a resource in order to use resource-specific consent queries. Learn more here.",
+            "Add": "ADD"
+        },
+
+        "Connected Resources": {
+            "Teams Channel ID": "Teams Channel ID",
+            "Chat": "Chat",
+            "Chat ID": "Chat ID",
+            "No Connected Resources": "No connected resources. Please connect to a Teams or Chats instance."
```

Comment:
In this message are you referring to Teams the product, or a team? Suggested revision: "You must connect to an instance of a team or a chat to make this request."